packages/gsm8k/src/aviary/envs/gsm8k/env.py

```diff
@@ -5,12 +5,12 @@
 from typing import TYPE_CHECKING, ClassVar, Literal
 
 import datasets
+from llmclient.messages import Message
```

Comment:
Could we make this `from llmclient import Message`? This looks like it'll be an extremely common import, so I think it'd help to keep it terse.